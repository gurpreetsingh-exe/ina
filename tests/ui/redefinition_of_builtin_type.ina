// COMPILE_FAIL

type i32 = {}

fn main() {}
