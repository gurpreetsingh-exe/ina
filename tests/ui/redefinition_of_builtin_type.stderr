error: redefinition of builtin type
   ╭─[tests/ui/redefinition_of_builtin_type.ina:3:6]
   │
 3 │ type i32 = {}
   │      ─┬─
   │       ╰── cannot define type with name `i32`, as it is a builtin type
   ∙
───╯
